enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -83,12 +83,12 @@ You have the following options:
 Start the Metabase container:
 
 ```bash
-docker run -d -p 3000:3000 --name metabase metabase/metabase-enterprise:v1.51.1
+docker run -d -p 3000:3000 --name metabase metabase/metabase-enterprise:latest
```

Comment:
These [tags are set manually](https://github.com/metabase/metabase/actions/workflows/release-tag.yml) every time we deploy a new version.

Both the Docker image tag and upload jar are updated roughly at the same time.

https://metaboat.slack.com/archives/C05MPF0TM3L/p1730131079713599?thread_ts=1730102593.927909&cid=C05MPF0TM3L